browser/views.py

```diff
@@ -104,6 +104,12 @@ def person(request, person_id=None):
         context['persons'] = PersonFilter(request.GET, queryset=Person.objects.order_by('last_name').filter(merge_from=None))
         # context['persons'] = get_paginator(Person, request, 'last_name', 100)
         template = "browser/person_list.html"
+
+    if request.method == 'POST':
```

Comment:
this should only be allowed if the user is a staff member

Reply:
I have added a code in the html file such that only users with staff privileges can see the delete and edit options for investigations. So, only staff users will be able to click on delete button and make the delete post request. 
Is this fine or there should be user validations in the <b>person</b> method in <b>views.py</b> file ?

Reply:
templates (which generates pages for the client side) and server api are two different things. just because you don't show the button in the browser, doesn't mean the api method isn't open to anyone (the browser is just making the request, another app could make a request, the user could modify the webpage before submitting, etc.).